### ISSUE TITLE
 __$$styleInject is not defined

### ISSUE THREAD

**Issue by pinguxx** [egoist/rollup-plugin-postcss#58]
after updating to 1.0.3 im getting that error, my config is
```
 postcss({
            plugins: [
                cssnano()
            ],
            extensions: ['.css']
        }),
```

**Comment by egoist** [comment]
can you provide a repro?

**Comment by egoist** [comment]
Fixed in `v1.0.4`

**Comment by pinguxx** [comment]
working, thanks
